syntax = "proto3";

package distributedcolony;

// PING
message PingRequest {
    string client_id = 1;
}

message PingResponse {
    int32 status = 1;
} 

// InitColony
message InitColonyRequest {
    int32 width = 1;
    int32 height = 2;
}

message InitColonyResponse {
    int32 status = 1;
} 